framework/Runners/__init__.py

```diff
@@ -25,7 +25,7 @@
 ## 'from Runners.Runner import Runner' outside
 ## of this submodule
 from .Runner import Runner
-from .ExternalRunner import ExternalRunner
+# from .ExternalRunner import ExternalRunner
```

Comment:
This doesn't exist anymore, right?  Is it a good idea to remove the commented code?